### PR TITLE
Updating/Adding github templates, updating CODEOWNERS file.

### PR DESCRIPTION
**Pull request by vrdmr**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+### Github basics
+
+#### GitHub workflow
+
+If you don't have experience with Git and Github, some of the terminology and process can be confusing. [Here's a guide to understanding Github](https://guides.github.com/introduction/flow/).
+
+#### Forking the Azure/Guest-Configuration-Extension repository
+
+Unless you are working with multiple contributors on the same file, we ask that you fork the repository and submit your Pull Request from there. [Here's a guide to forks in Github](https://guides.github.com/activities/forking/).
```
Comment (Hazhzeng) [inline .github/CONTRIBUTING.md:29]: We should mention that we **block** PRs from forks to go through E2E tests. However, we will cherry-picked PRs once they pass unit tests.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Contributing to Linux Guest Agent
```
Comment (Hazhzeng) [inline .github/CONTRIBUTING.md:1]: This is Python worker library, not **Linux Guest Agent**

Comment (Hazhzeng) [inline .github/CONTRIBUTING.md:1]: This is **Azure Functions Python Library**, not Linux Guest Agent

Comment (vrdmr) [reply .github/CONTRIBUTING.md:1]: Good catch. I picked up from the template. I'll update this.

**Pull request merged**